src/app/page.tsx

```diff
@@ -3,7 +3,11 @@ import Link from "next/link";
 import { LatestPost } from "~/app/_components/post";
 import { getServerAuthSession } from "~/server/auth";
 import { api, HydrateClient } from "~/trpc/server";
+
+// Components
 import { UserProfile } from "./_components/userProfile/UserProfile";
+import SignInButton from "./_components/NextAuth/SignInButton";
+import SignOutButton from "./_components/NextAuth/SignOutButton";
```

Comment:
Same here, prefer path aliased absolute imports and named imports over relative imports and default imports.